bot/utils/scheduling.py

```diff
@@ -157,18 +157,18 @@ def _task_done_callback(self, task_id: t.Hashable, done_task: asyncio.Task) -> N
                 self._log.error(f"Error in task #{task_id} {id(done_task)}!", exc_info=exception)
 
 
-def create_task(*args, **kwargs) -> asyncio.Task:
+def create_task(coro: t.Awaitable, *suppressed_exceptions: t.Type[Exception], **kwargs) -> asyncio.Task:
```

Comment:
Passing in the exceptions as positional args doesn't seem clear to me from the caller's side. 
A tuple kwarg would explain the uses of the exception types properly without a comment like the one in the messages module where this functionality is currently used